client/signup/config/flows.js

```diff
@@ -219,6 +219,11 @@ function filterFlowName( flowName ) {
 			return ( 'blog' === flowName ) ? 'blog-altthemes' : 'website-altthemes';
 		}
 	}
+
+	if ( 'skipForFree' === abtest( 'freePlansDefault' ) ) {
```

Comment:
Yes, see the `altThemes` test above for how to test for other variations and make sure you're not stepping on other tests. 

Also worth mentioning that this function (`filterFlowName`) is called at least twice during signup, so make sure it will return what you expect it to return once the new flow has been assigned. (Tangent: we should probably cache the return value of this function.)